libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -216,3 +216,8 @@
 .is-fixed-width {
   width: 10rem;
 }
+
+.is-close-btn {
+  width: 24px;
+  height: 24px;
+}
```

Comment:
I would use [spacing helpers](https://bulma.io/documentation/helpers/spacing-helpers/) directly on button class

Reply:
Should I add padding to the button based on the size of the icon?

Reply:
![Screenshot from 2023-08-29 14-14-27](https://github.com/kodadot/nft-gallery/assets/47194071/b92dbfa8-a11b-4ba8-9174-6aa2e284f028)
![Screenshot from 2023-08-29 14-14-22](https://github.com/kodadot/nft-gallery/assets/47194071/dd1ed673-f660-483c-a23b-9d65c8778045)
![Screenshot from 2023-08-29 14-14-18](https://github.com/kodadot/nft-gallery/assets/47194071/daa807c6-0f54-40e4-855f-35d2043cd222)